pkg/operator/ceph/object/spec.go

```diff
@@ -113,7 +113,7 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 			controller.DaemonVolumes(c.DataPathMap, rgwConfig.ResourceName),
 			c.mimeTypesVolume(),
 		),
-		HostNetwork:        c.clusterSpec.Network.IsHost(),
+		HostNetwork:        c.store.Spec.Network.IsHost(),
```

Comment:
You should add some unit test related to it

Reply:
done